Runtime/Core/AxisState.cs

```diff
@@ -424,6 +423,9 @@ public void RecenterNow()
             /// <param name="recenterTarget">The value that is considered to be centered</param>
             public void DoRecentering(ref AxisState axis, float deltaTime, float recenterTarget)
             {
+                deltaTime = Time.realtimeSinceStartup - m_LastUpdateTime;
```

Comment:
you need to comment why we're setting deltaTime even though the user passed it in.  Also, you need to add some checks (same as above).  You can't just blindly set this.  deltaTime < 0 has special meaning in CM.